lewis/adapters/stream.py

```diff
@@ -241,20 +328,26 @@ def _parseArguments(self, arguments):
     def _create_properties(self, cmds):
         patterns = set()
         for cmd in cmds:
-            method = cmd.method
+            member = cmd.member
+
+            if member not in dir(self):
+                if member not in dir(self._device):
+                    raise AttributeError('Can not find member \''
+                                         + member + '\' in device or interface.')
+
+                if callable(getattr(self._device, member)):
+                    setattr(self, member, ForwardMethod(self._device, member))
+                else:
+                    setattr(type(self), member, ForwardProperty('_device', member, instance=self))
```

Comment:
This is the whole data descriptor thing...on the object it looks like an attribute, so you can never tell from the object whether it's that or a `property`. I'm not sure my understanding is good enough to explain it in a proper way...

But I do think `ForwardProperty` could be implemented in a different way, more like in `control_client`.